addon/src/-internal/build-registry.ts

```diff
@@ -102,10 +99,9 @@ export default function buildRegistry(resolver: Resolver) {
   const fallbackRegistry = Application.buildRegistry(namespace);
   // TODO: only do this on Ember < 3.13
```

Comment:
Is this comment relevant? If so, do we even need to be supporting Ember < 3.13 here?

Reply:
We don't, but I'd argue cleanup for legacy stuff should be in a different PR (which I can submit next! as there are a number of other cleanup opportunities, I think)